storage/oasis/nodeapi/api.go

```diff
@@ -131,29 +131,29 @@ type Event struct {
 	// Called "Kind" in oasis-core but "Type" in Nexus APIs and DBs.
 	Type apiTypes.ConsensusEventType
 
-	StakingTransfer             *TransferEvent
-	StakingBurn                 *BurnEvent
-	StakingAddEscrow            *AddEscrowEvent
-	StakingTakeEscrow           *TakeEscrowEvent
-	StakingEscrowDebondingStart *DebondingStartEscrowEvent
-	StakingReclaimEscrow        *ReclaimEscrowEvent
-	StakingDebondingStart       *DebondingStartEscrowEvent // Available starting in Damask.
-	StakingAllowanceChange      *AllowanceChangeEvent
-
-	RegistryRuntimeStarted   *RuntimeStartedEvent
-	RegistryRuntimeSuspended *RuntimeSuspendedEvent // Available starting with Eden.
-	RegistryEntity           *EntityEvent
-	RegistryNode             *NodeEvent
-	RegistryNodeUnfrozen     *NodeUnfrozenEvent
-
-	RoothashMisc              *RoothashEvent
-	RoothashExecutorCommitted *ExecutorCommittedEvent
-	RoothashMessage           *MessageEvent // Available only in Cobalt.
-
-	GovernanceProposalSubmitted *ProposalSubmittedEvent
-	GovernanceProposalExecuted  *ProposalExecutedEvent
-	GovernanceProposalFinalized *ProposalFinalizedEvent
-	GovernanceVote              *VoteEvent
+	StakingTransfer             *TransferEvent             `json:",omitempty"`
```

Comment:
It looks like we removed a lot of these in https://github.com/oasisprotocol/nexus/commit/02df003e352aaa58b5faabf6c4a6d9394e7107b9 (by mistake?).

Reply:
@Andrew7234 how did those json annotations come up in the api tests PR?